src/main/java/fr/insalyon/creatis/gasw/plugin/executor/dirac/execution/DiracJdlGenerator.java

```diff
@@ -78,12 +79,14 @@ private DiracJdlGenerator() throws GaswException {
         this.priority = conf.getDefaultPriority();
         this.site = "";
         this.bannedSites = new StringBuilder();
+        this.defaultBannedSites = new StringBuilder();
         for (String bSite : DiracConfiguration.getInstance().getBannedSites()) {
-            if (bannedSites.length() > 0) {
-                this.bannedSites.append(",");
+            if (defaultBannedSites.length() > 0) {
+                this.defaultBannedSites.append(",");
             }
-            this.bannedSites.append(bSite);
+            this.defaultBannedSites.append(bSite);
         }
+        this.bannedSites = this.bannedSites.append(this.defaultBannedSites);
```

Comment:
`StringBuilder.append` returns itself, so its not necessary to store the results in `this.bannedSites` : it's the same Stringbuilder that has already changed.